tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -122,20 +122,18 @@ public function userCreatesFileInsideFolderInSpaceUsingWopiEndpoint(string $user
 	}
 
 	/**
-	 * @When the public creates a file :file inside the last shared public link folder with password :password using wopi endpoint
-	 * @When the public tries to create a file :file inside the last shared public link folder with password :password using wopi endpoint
-	 *
 	 * @param string $file
 	 * @param string $password
+	 * @param string|null $folder
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function thePublicCreatesAFileInsideTheLastSharedPublicLinkFolderWithPasswordUsingWopiEndpoint(string $file, string $password): void {
+	public function createFile(string $file, string $password, ?string $folder = null): void {
 		$token = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
 		$davPath = WebDavHelper::getDavPath($token, null, "public-files-new");
```

Comment:
directly appending null might result in failure.
```suggestion
	public function createFile(string $file, string $password, ?string $folder = ""): void {
		$token = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
		$davPath = WebDavHelper::getDavPath($token, null, "public-files-new") . "/$folder";
```